libs/build-nss-desktop.sh

```diff
@@ -33,6 +33,13 @@ elif [[ -n "${CROSS_COMPILE_TARGET}" ]]; then
 elif [[ "$(uname -s)" == "Darwin" ]]; then
   DIST_DIR=$(abspath "desktop/darwin/nss")
   TARGET_OS="macos"
+  #We need this variable for switching libs based on different macos archs (M1 vs Intel)
+  #also, keeping the naming consistent with target_arch env on the rust side
```

Comment:
I think we should s/target_arch/CARGO_CFG_TARGET_ARCH/ to make it extra clear what we are trying to do